source/reference/kubectl-minio-plugin/kubectl-minio-init.rst

```diff
@@ -83,7 +83,7 @@ The command supports the following flags:
    :optional:
 
    The image to use when deploying the :minio-git:`MinIO Console <console>` in Operator mode, where administrators can create and manage MinIO tenants using a Graphical User Interface.
-   Defaults to ``minio/console:v0.17.3``.
+   Defaults to the :minio-git:`latest release of the operator <operator/releases/latest>`.
```

Comment:
```suggestion
   Defaults to the version bundled with the `operator release version <operator/releases/latest>`__ matching the plugin version.
```

Since someone could use this doc with an older `kubectl minio` version, and the console version would be whatever is attached to that version.

I don't think it's necessary to keep a matrix, just mention they are bundled.

Reply:
mentioned the variable that holds the value